src/components/Footer.tsx

```diff
@@ -5,17 +5,39 @@ const Bar = styled.div`
   height: 2rem;
   width: 95%;
   align-items: center;
-  color: rgba(150, 150, 150, 1);
+  color: rgba(80, 80, 80, 1);
   text-align: left;
   font-family: Barlow, sans-serif;
   font-size: 11px;
   font-style: italic;
   padding-left: 1rem;
+  padding-bottom: 1em;
 `;
 
-const Footer: FC = () => {
+const Source = styled.div`
+  padding-bottom: 1em;
+`;
+
+export function getSourceType(src: string) {
+  switch (src) {
+    case 'NIELSEN':
+      return 'Nielsen BookData';
+    case 'BOKBASEN':
+      return 'Bokbasen';
+    default:
+      //ALMA, ADABAS, BIBSYS
+      return 'Unit';
+  }
+}
+
+interface FooterProps {
+  source: string;
+}
+
+const Footer: FC<FooterProps> = (props) => {
```

Comment:
istedenfor å bruke props, kan man dekomponere  `= ({source}) = {`

Reply:
Den skjønner jeg ikke.
`const Footer: FC<FooterProps> = ({source}) = {` 
fører til at alt lyser opp som en juletre med feilmeldinger.
Show me the way, Master!

Reply:
`const Footer: FC<FooterProps> = ({source}) => {`